deltaPostProcess.js

```diff
@@ -0,0 +1,17 @@
+import { isPublishedService, removePublishedStatus } from './queries';
+
+export async function processDelta(deltabody){
+  const admsStatus = "http://www.w3.org/ns/adms#status";
+  const sent_uri = "http://lblod.data.gift/concepts/9bd8d86d-bb10-4456-a84e-91e9507c374c";
```

Comment:
@benjay10 @jsz4n I am not sure this is the filter we need.
The flow we need to tackle; if a a thing is status 'sent' and 'published', but the user re-opens it by setting it to 'concept'. This triggers and insert query and the state in the DB will be published. Then only we need to remove the published flag.